config/routes.rb

```diff
@@ -1,3 +1,7 @@
 Rails.application.routes.draw do
-  # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
+  root 'parties#index'
+
+  get '/login', to: 'sessions#new', as: :login
+  get '/auth/spotify', as: :spotify_oath
```

Comment:
This may seem nitpicky, but I could envision a scenario where someone would type oauth instead of oath... maybe this is a typo?

Reply:
> This may seem nitpicky, but I could envision a scenario where someone would type oauth instead of oath... maybe this is a typo?

these are the routes that OmniAuth uses to build the callback URIs, so unfortunately this is how they have to be written in the routes.

Reply:
It’s 100% a typo on the alias for the path helper. I’ll push a correction in both locations as soon as I’m home.

Reply:
Disregard my comment. I thought you were referring to the route /auth not the path helper in this comment! Yes that is a typo!

Reply:
We want them to swear an oath in order to use our product